libraries/Parsers/Microsoft.Bot.Builder.Parsers.LU/Parser/LUResource.cs

```diff
@@ -0,0 +1,407 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This class has me a bit confused. It's got a number of warnings disables, and it's violating our "1 class per file" rule. While I think this is probably a bad idea, please at least put a detailed comment in the file explaining why you're doing this. 
My strong preference is to fix all of this.

Reply:
This was auto-generated, we can discuss taking it and applying all the necessary refactor but we were scoping that for next version